doc/api/n-api.md

```diff
@@ -4920,7 +4920,7 @@ napiVersion: 2
 
 ```C
 NAPI_EXTERN napi_status napi_get_uv_event_loop(napi_env env,
-                                               uv_loop_t** loop);
+                                               uv_loop_s** loop);
```

Comment:
```suggestion
                                               struct uv_loop_s** loop);
```

Reply:
IIUC, `uv_loop_t` is identical to `struct uv_loop_s` technically right?

Reply:
Yes and I think that it is used because if you import N-API header (`node_api.h`) you don't have `uv_loop_t`.  If you will use `uv_loop_t` inside your code the compiler complains saying that is not possible to determine the type `uv_loop_t`.